src/main/java/com/bunq/sdk/context/SessionContext.java

```diff
@@ -101,5 +101,4 @@ Date getExpiryTime() {
   public Integer getUserId() {
     return userId;
   }
-
```

Comment:
please revert the removal of this line.

Reply:
bump

Reply:
please revert the removal of this line. As described here: https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace